.storybook/guides/develop.mdx

```diff
@@ -125,6 +127,54 @@ Notes on a few commonly used argTypes:
 
 - If your component does not include any animation, please hide the reducedMotion control to avoid confusion: `reducedMotion: { table: { disable: true } }`
 
+u can import state-based argTypes that are commonly used from the `types/states.js`. States available in this file include (but are not limited to):
```

Comment:
😅 did this line get chopped off at the start of the sentence? Maybe `You` instead of `u`?

Reply:
haha!  No I'm trying to be hip with the gen Zs. 😉